sippy-ng/src/component_readiness/CompReadyMainInputs.js

```diff
@@ -156,3 +156,63 @@ export default function CompReadyMainInputs(props) {
 
 // component and environment may be null so they are not required
 CompReadyMainInputs.propTypes = {}
+
+export function CompReadyTestDetailInputs(props) {
```

Comment:
might belong in a different file?

Reply:
Yea if it's big enough usually it should go in a file by itself - do you share any code between the two that means they need to stay together?